uptime-kuma/Dockerfile

```diff
@@ -48,6 +48,8 @@ RUN \
         /root/.npm \
         /root/.npmrc
 
+RUN curl -L https://github.com/cloudflare/cloudflared/releases/latest/download/cloudflared-linux-amd64 -o /usr/bin/cloudflared \
+    && chmod +x /usr/bin/cloudflared
```

Comment:
Hi there @samuelleb11 👋 

Two things:

- We don't have to create an additional Docker layer, just add it to the sequence above
- The `cloudflared` ain't versioned. I rather have a version locked (so builds are reproducible, caches work properly, and we can ship updates if needed).

For the latter, please add an entry to the renovate configuration as well. We can pick up the latest information from: <https://github.com/cloudflare/cloudflared/releases>